### DISCUSSION TITLE
Tout ce que je sais sur RxJS | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#10] [General]
# Tout ce que je sais sur RxJS | Bonjour Angular

Tout ce que je sais sur RxJS

https://apprendre.bonjour-angular.com/articles/tout-ce-que-je-sais-sur-rxjs/

**Comment by nathandubocage** [comment]
Merci beaucoup pour ce partage d'expérience ! J'ai découvert bonjour-angular aujourd'hui et c'est top d'avoir du contenu Français de qualité sur Angular :)

**Comment by ecolinet-orange** [comment]
Bonjour,
Merci pour ces infos très utiles.
Sauf erreur de ma part il me semble que le code suivant contient une coquille :
```ts
const obs = new Observable((subscriber) => {
  try {
    subscriber.next(Math.random());
    subscriber.complete();
  } catch (error) {
    throw new Error(error); // <== ICI
  }
});

obs.subscribe({
  next: (data) => console.log(data),
  error: (error) => console.error(error),
  complete: () => console.log("complete"),
});
```

Cela devrait être `subscriber.error(error.message)` pour que ce soit bien récupéré dans le call back error.
